packages/node/src/runtimePlugin.ts

```diff
@@ -131,7 +131,8 @@ export default function () {
             };
             return callback(null, emptyChunk);
           }
-          fetch(url)
+          const fetchFunction = globalThis.webpackChunkLoad || fetch;
```

Comment:
https://github.com/module-federation/core/blob/05c43f3e7ef1e7fc33638ecb27bbb422086810a8/packages/runtime/src/core.ts#L105

Maybe we can set fetch through fetch hook instead of using global variables

Reply:
@zhoushaw do you have a sample of how to use the fetch hook? 
I do not think it is bound to script loader, only json manifest loader

This provides backward compat for user, but i am happy to look at better solution for future and deprecate this usage